erc20-load-test-tool/src/main/java/com/vmware/ethereum/config/Web3jConfig.java

```diff
@@ -70,5 +70,6 @@ public static class Receipt {
     @NotNull private long interval;
     @NotNull private boolean defer;
     @NotNull private boolean checkWritetxFailed;
+    @Positive private int retryWritetxSleep;
```

Comment:
I think Write and Tx should be treated as separate words.

Reply:
Changed